MetaMask/metamask-extension
# feat: update NFT approve page title

## **Description**

Change NFT Approve title from `Allowance Request` to `Withdrawal Request`.

## **Related issues**

Fixes: NA

## **Manual testing steps**

1. Open NFT allowance confirmation
2. Ensure that title is `Withdrawal Request`

## **Screenshots/Recordings**
TODO

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
